cmd/generate_test.go

```diff
@@ -32,6 +32,7 @@ func TestMain(t *testing.T) {
 		c := bytes.NewBufferString("")
 		cmd.SetArgs(args)
 		cmd.SetErr(c)
+		cmd.SetOut(bytes.NewBufferString(""))
```

Comment:
Did this all in the error-path generate tests to cut down on the useless output (the "Usage" instructions Cobra prints by default when there is an error)